DS13/code/modules/borg/borg.dm

```diff
@@ -0,0 +1,323 @@
+GLOBAL_DATUM_INIT(borg_collective, /datum/borg_collective, new)
+
+/datum/borg_collective
+	var/name = "Collective node 01"
+	var/list/drones = list() //Who's a borg? You're a borg!
+	var/last_alert = 0 //At what %age completion did we last alert our drones? Do we value our drones' hearing?
+	var/borg_whisper_chance = 30 //This will need changing
+	var/adaptation = 0 //how adapted are they out of 100
+
+/datum/borg_collective/proc/message_collective(var/who, var/what) //Both as strings, please!
+	if(!who)
+		who = "Unknown"
+	for(var/mob/living/S in drones)
```

Comment:
Change it to human. We dont plan to make Ian a borg since the code wont work properly for it.